x-pack/legacy/plugins/integrations_manager/server/packages/install.ts

```diff
@@ -45,7 +45,7 @@ export async function installAssets(options: {
 
   // Only install certain Kibana assets during package installation.
   // All other asset types need special handling
-  const typesToInstall = [AssetType.visualization, AssetType.dashboard, AssetType.search];
```

Comment:
interesting that this didn't cause a crash but mine did...

Reply:
I think _accessing_ the enum is what caused the crash. This is removing them.

Reply:
Yes, I meant before they were removed.